src/som/interpreter/actors/EventualMessage.java

```diff
@@ -269,4 +272,14 @@ public static EventualMessage getCurrentExecutingMessage() {
     Thread t = Thread.currentThread();
     return ((ActorProcessingThread) t).currentMessage;
   }
+
+  //indicates if the message is already in the actor mailbox and it is paused (due to breakpoint or step into command)
+  public boolean hasPause() {
+    return pause;
+  }
+
+  //call when adding a breakpoint
+  public void setPause(final boolean value) {
```

Comment:
Is this really needed, can't we make the field final and properly initialize the message on creation?